assets/js/src/tracker/data-formatting.js

```diff
@@ -178,10 +178,14 @@ export const purchase = ( { order } ) => {
 	}
 
 	return {
-		currency: order.totals.currency_code,
-		value: formatPrice(
-			order.totals.total_price,
-			order.totals.currency_minor_unit
+		transaction_id: order.id,
+		affiliation: order.affiliation,
+		currency: order.currency_code,
+		value: formatPrice( order.total_price, order.currency_minor_unit ),
+		tax: formatPrice( order.tax_total, order.currency_minor_unit ),
+		shipping: formatPrice(
+			order.shipping_total,
+			order.currency_minor_unit
```

Comment:
Maybe we could keep the same structure as we have for cart, and close to what WC Core uses:
```suggestion
		currency: order.totals.currency_code,
		value: formatPrice(
			order.totals.total_price,
			order.totals.currency_minor_unit
		),
		tax: formatPrice(
			order.totals.tax_total,
			order.totals.currency_minor_unit
		),
		shipping: formatPrice(
			order.totals.shipping_total,
			order.totals.currency_minor_unit
```